lib/funky/post.rb

```diff
@@ -0,0 +1,19 @@
+module Funky
+  class Post < GraphRootNode
+    # @return [String] the type of post.
+    def type
+      data[:type]
+    end
+
+    # @return [DateTime] the created time of the post.
+    def created_time
+      datetime = data[:created_time]
+      DateTime.parse datetime if datetime
+    end
+
+    # @return [Integer] the number of shares of the post.
+    def count_shares
+      data[:shares] && data[:shares][:count] || 0
```

Comment:
@kangkyu Why these conditions and not just `data[:shares][:count]`?

Did you find any post that does not have `[:shares]` or does not have `[:shares][:count]` ?